conda/recipes/cudf/meta.yaml

```diff
@@ -67,16 +78,23 @@ requirements:
     - numba >=0.57
     - numpy >=1.21
     - {{ pin_compatible('pyarrow', max_pin='x.x.x') }}
-    - libcudf {{ version }}
+    - libcudf ={{ version }}
     - {{ pin_compatible('rmm', max_pin='x.x') }}
     - fsspec >=0.6.0
+    {% if cuda_major == "11" %}
     - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
-    - nvtx >=0.2.1
-    - packaging
     - ptxcompiler >=0.7.0
-    - cachetools
     - cubinlinker  # CUDA enhanced compatibility.
     - cuda-python >=11.7.1,<12.0
+    {% else %}
+    - cuda-nvrtc
+    - libcufile  # [linux64]
```

Comment:
numba requires nvrtc, yeah.  I agree that libcufile can be removed here, it's not a direct dependency of cudf. I assume it'll always be pulled transitively via libcudf, but that's the right way it should work.

Reply:
Also should note that numba won't pull nvrtc automatically IIRC because you could install numba without CUDA support. It only has `run_constrained` in place (and I don't know if that has been updated for CUDA 12, so it might just have a `run_constrained` on the `cudatoolkit` package.